c/tskit/tables.h

```diff
@@ -697,28 +697,21 @@ typedef struct {
 /* Leave room for more positive check flags */
 #define TSK_NO_CHECK_POPULATION_REFS (1 << 12)
 
-/* Flags for load tables */
-#define TSK_BUILD_INDEXES (1 << 0)
+/* Flags for table collection load or init
+ * As flags are passed though from load to init they share a namespace */
+#define TSK_LOAD_INIT_NO_METADATA (1 << 0)
+#define TSK_LOAD_INIT_SKIP_TABLES (1 << 1)
+#define TSK_LOAD_INIT_SKIP_REFERENCE_SEQUENCE (1 << 2)
+#define TSK_LOAD_INIT_NO_EDGE_METADATA (1 << 3)
+#define TSK_LOAD_INIT_BUILD_INDEXES (1 << 4)
```

Comment:
This one is different, in that it's only ever passed to tsk_treeseq_init(), right?

Reply:
Could call ``TSK_TS_INIT_BUILD_INDEXES``